aspire_cloud_hosting_policy.md

```diff
@@ -0,0 +1,53 @@
+# Aspire Cloud Hosting Policy
+
+## Preamble
+
+The goal of Aspire Cloud Repository is to provide a robust distributed source for downloading software for extending WordPress and related compatible CMSs. The Distributed Software Repository project is released freely in order to establish a network of trusted repositories which democratize software distribution free from the control of any one person or organization.
+
+This DRAFT policy governs what software will be accepted to host on AspirePress Cloud. At the project's inception, we are providing a general list of requirements and deferring to the corresponding [guidelines available at wordpress.org](https://developer.wordpress.org/plugins/wordpress-org/detailed-plugin-guidelines/) for interpretation until they have been fully rewritten in this document. While this document is in draft form, in case of conflicts or updates, the guideline explanations below will supersede any others.
+
+If you believe you have a case where a software package would not be accepted for hosting on wordpress.org, but believe it should be allowed on Aspire Cloud, please contact support@aspirepress.org.
+
+While this Preamble should help understand and interpret the Hosting Policy, it is not a formal part of Policy.
+
+-----
+
+## Aspire Cloud Guidelines, Terms, & Conditions
+
+### Licensing & Other Requirements
+
+1. Plugins must be compatible with the GNU General Public License v2, but may use any license which qualifies.
+2. By submitting your software and updates, you agree to indemnify and hold harmless AspirePress and its network partners for any actions resulting from your software.
+3. Code must be (mostly) human readable.
+4. Developers are responsible for the contents and actions of their plugins.
+5. A stable version of a plugin must be available from its WordPress Plugin Directory page.
+6. Plugins must respect trademarks, copyrights, and project names.
+7. Developers and their plugins must not do anything illegal, dishonest, or morally offensive.
+8. A complete plugin must be available at the time of submission.
+
+### Features & Functions Not Permitted
+
+8. Trialware is not permitted.
```

Comment:
Let's expand on this, maybe add "Plugins may not disable any of their own functionality after any trial period has ended.  Any plugin offering limited-time functionality must be distributed through other channels."

We might still want to offer that other channel ourselves, we could be an app store for paid plugins at some point, but I think that's far in the future, and I'd definitely want that business walled off from the main organization.